File: app/src/main/java/com/tutv/android/datasource/dto/SeriesFollowedResponseDTO.java
package com.tutv.android.datasource.dto;

public class SeriesFollowedResponseDTO {

    private int followers;
    private boolean loggedInUserFollows;

    public SeriesFollowedResponseDTO() {}

    public SeriesFollowedResponseDTO(int followers, boolean loggedInUserFollows) {
        this.followers = followers;
        this.loggedInUserFollows = loggedInUserFollows;
    }

    public int getFollowers() {
        return followers;
    }

    public void setFollowers(int followers) {
        this.followers = followers;
    }

    public boolean getLoggedInUserFollows() {
        return loggedInUserFollows;
    }

    public void setLoggedInUserFollows(Boolean loggedInUserFollows) {
        this.loggedInUserFollows = loggedInUserFollows;
    }
}

File: app/src/main/java/com/tutv/android/datasource/dto/SeriesFollowedResponseDTO.kt
package com.tutv.android.datasource.dto

import org.mockito.ArgumentMatchers
import org.mockito.InOrder
import org.mockito.Mockito

class SeriesFollowedResponseDTO {
    private var followers = 0
    private var loggedInUserFollows = false

    constructor() {}
    constructor(followers: Int, loggedInUserFollows: Boolean) {
        this.followers = followers
        this.loggedInUserFollows = loggedInUserFollows
    }

    fun getFollowers(): Int {
        return followers
    }

    fun setFollowers(followers: Int) {
        this.followers = followers
    }

    fun getLoggedInUserFollows(): Boolean {
        return loggedInUserFollows
    }

    fun setLoggedInUserFollows(loggedInUserFollows: Boolean?) {
        this.loggedInUserFollows = loggedInUserFollows
    }
}